### PR TITLE
initial commit for camera image cache

### PR DESCRIPTION
**Pull request by birdage**
@FBRTMaka @DanielJMaher <b>do not merge </b>

* checks for existing
* adds config option for location

### CODE REVIEW

**Comment by DanielJMaher** [comment]
@birdage you failed

**Comment by DanielJMaher** [comment]
INFO in __init__ [/home/travis/build/asascience-open/ooi-ui-services/ooiservices/app/__init__.py:65]:
Application Process Started
--------------------------------------------------------------------------------
Traceback (most recent call last):
  File "ooiservices/manage.py", line 30, in <module>
    app = create_app(env)
  File "/home/travis/build/asascience-open/ooi-ui-services/ooiservices/app/__init__.py", line 83, in create_app
    from ooiservices.app.main import api as main_blueprint
  File "/home/travis/build/asascience-open/ooi-ui-services/ooiservices/app/main/__init__.py", line 13, in <module>
    from ooiservices.app.main import routes, authentication, user, operator_event, annotation, instrument_deployment, \
  File "/home/travis/build/asascience-open/ooi-ui-services/ooiservices/app/main/alertsalarms.py", line 16, in <module>
    from ooiservices.app.uframe.assets import get_assets
  File "/home/travis/build/asascience-open/ooi-ui-services/ooiservices/app/uframe/__init__.py", line 13, in <module>
    from ooiservices.app.uframe import controller, assetController, data, assets, events, subscribe
  File "/home/travis/build/asascience-open/ooi-ui-services/ooiservices/app/uframe/controller.py", line 35, in <module>
    from bs4 import BeautifulSoup
ImportError: No module named bs4
The command "python ooiservices/manage.py deploy -p test" failed and exited with 1 during .

**Comment by birdage** [comment]
@DanielJMaher ill fix up the conflicts and work with maka to make sure its ready to go when we need it.

**Comment by DanielJMaher** [comment]
@FBRTMaka HOlding unti lUI side is clear. Merge after you are comfortable with both.  Thanks.

File: ooiservices/app/celeryconfig.py [hunk 1]
```diff
@@ -21,4 +21,9 @@
         'schedule': crontab(hour='*/8'),
         'args': (),
         },
+    'get-cam-images-every': {
+        'task': 'tasks.compile_cam_images',
+        'schedule': crontab(hour='*/12'),
```
Comment (FBRTMaka) [inline ooiservices/app/celeryconfig.py]: @birdage we've updated this code overall, but you'll need to update this as well to be:

crontab(minute=0, hour='*/12'),

for some reason celery doesn't like omitting the minutes, even if it's 0

**Comment by FBRTMaka** [comment]
@birdage nice, I can't see anything else and I've tested on a linux environment.  Looks sweet!

:+1:

**Pull request merged**